python-opendata/opendata/files.py

```diff
@@ -0,0 +1,37 @@
+import glob
+from os import listdir, path, walk
+
+import pandas as pd
+
+
+def downloaded_athletes():
+    athlete_ids = []
+    for f in listdir(path.join('.opendata_storage')):
+        if path.isdir(path.join('.opendata_storage', f)):
+            athlete_ids.append(f)
+
+    return athlete_ids
+
+
+def activities_for_athlete(athlete_id):
+    activity_ids = []
```

Comment:
list comprehension

Reply:
It's a matter of preference but I really despise list comprehensions that need line wrapping... 😅 for-loops are far easier to read than long list comprehensions in my opinion and there are no real drawbacks.